README.md

```diff
@@ -0,0 +1,188 @@
+# torchvision
+
+[![total torchvision downloads](https://pepy.tech/badge/torchvision)](https://pepy.tech/project/torchvision)
+[![documentation](https://img.shields.io/badge/dynamic/json.svg?label=docs&url=https%3A%2F%2Fpypi.org%2Fpypi%2Ftorchvision%2Fjson&query=%24.info.version&colorB=brightgreen&prefix=v)](https://pytorch.org/vision/stable/index.html)
+
+The torchvision package consists of popular datasets, model architectures, and common image transformations for computer
+vision.
+
+## Installation
+
+We recommend Anaconda as Python package management system. Please refer to [pytorch.org](https://pytorch.org/) for the
+detail of PyTorch (`torch`) installation. The following is the corresponding `torchvision` versions and supported Python
+versions.
+
+| `torch`            | `torchvision`      | Python              |
+| ------------------ | ------------------ | ------------------- |
+| `main` / `nightly` | `main` / `nightly` | `>=3.8`, `<=3.11`   |
+| `2.0.0`            | `0.15.1`           | `>=3.8`, `<=3.11`   |
+| `1.13.0`           | `0.14.0`           | `>=3.7.2`, `<=3.10` |
+| `1.12.0`           | `0.13.0`           | `>=3.7`, `<=3.10`   |
+| `1.11.0`           | `0.12.0`           | `>=3.7`, `<=3.10`   |
+
+<details>
+    <summary>older versions</summary>
+
+    | `torch`            | `torchvision`      | Python                    |
+    |--------------------|--------------------|---------------------------|
+    | `1.10.2`           | `0.11.3`           | `>=3.6`, `<=3.9`          |
+    | `1.10.1`           | `0.11.2`           | `>=3.6`, `<=3.9`          |
+    | `1.10.0`           | `0.11.1`           | `>=3.6`, `<=3.9`          |
+    | `1.9.1`            | `0.10.1`           | `>=3.6`, `<=3.9`          |
+    | `1.9.0`            | `0.10.0`           | `>=3.6`, `<=3.9`          |
+    | `1.8.2`            | `0.9.2`            | `>=3.6`, `<=3.9`          |
+    | `1.8.1`            | `0.9.1`            | `>=3.6`, `<=3.9`          |
+    | `1.8.0`            | `0.9.0`            | `>=3.6`, `<=3.9`          |
+    | `1.7.1`            | `0.8.2`            | `>=3.6`, `<=3.9`          |
+    | `1.7.0`            | `0.8.1`            | `>=3.6`, `<=3.8`          |
+    | `1.7.0`            | `0.8.0`            | `>=3.6`, `<=3.8`          |
+    | `1.6.0`            | `0.7.0`            | `>=3.6`, `<=3.8`          |
+    | `1.5.1`            | `0.6.1`            | `>=3.5`, `<=3.8`          |
+    | `1.5.0`            | `0.6.0`            | `>=3.5`, `<=3.8`          |
+    | `1.4.0`            | `0.5.0`            | `==2.7`, `>=3.5`, `<=3.8` |
+    | `1.3.1`            | `0.4.2`            | `==2.7`, `>=3.5`, `<=3.7` |
+    | `1.3.0`            | `0.4.1`            | `==2.7`, `>=3.5`, `<=3.7` |
+    | `1.2.0`            | `0.4.0`            | `==2.7`, `>=3.5`, `<=3.7` |
+    | `1.1.0`            | `0.3.0`            | `==2.7`, `>=3.5`, `<=3.7` |
+    | `<=1.0.1`          | `0.2.2`            | `==2.7`, `>=3.5`, `<=3.7` |
```

Comment:
This is rendered as a code with 4 spaces indent

Reply:
Awesome, thanks!